INSTALL.md

```diff
@@ -52,6 +52,21 @@ kafka/bin/zookeeper-server-start.sh -daemon kafka/config/zookeeper.properties
 kafka/bin/kafka-server-start.sh -daemon kafka/config/server.properties
 ```
 
+On MacOS, you can use [homebrew](https://brew.sh/) for easier setup of kafka
+
+```bash
+brew install kafka
+brew services start zookeeper
+brew services start kafka
+```
+
+You can stop Kafka service by running
```

Comment:
this won't work on non-MacOS!

Reply:
(isn't it kinda obvious anyway?)

Reply:
ah yes, I felt that while writing too 😋  But added it just in case someone doesn't know.
Will remove it!